composer.json

```diff
@@ -24,7 +24,11 @@
     "embed/embed": "^2.6"
   },
   "require-dev": {
-    "phpunit/PHPUnit": "~4.8"
+    "phpunit/PHPUnit": "~4.8",
+    "silverstripe/behat-extension": "dev-master",
+    "silverstripe/serve": "dev-master",
+    "silverstripe/testsession": "dev-master",
```

Comment:
I've locked down behat and testsession but I'm not sufficiently confident that I'm done tinkering with serve and would rather lock that down after I've got the cms build working.